mage_integrations/mage_integrations/destinations/elasticsearch/target_elasticsearch/sinks.py

```diff
@@ -186,8 +186,7 @@ def authenticated_client(self) -> elasticsearch.Elasticsearch:
             scheme = "https"
             config["ca_certs"] = self.config[SSL_CA_FILE]
 
-        if scheme == "https":
-            config['verify_certs'] = self.config[VERIFY_CERTS]
+        self.config.get(VERIFY_CERTS, True)
```

Comment:
I guess you still want to set `config['verify_certs']`
like
```
if scheme == "https":
    config['verify_certs'] = self.config.get(VERIFY_CERTS, True)
```

Reply:
Sorry fixed this one